packages/components/psammead-radio-schedule/src/ProgramCard/index.jsx

```diff
@@ -0,0 +1,250 @@
+    fill: currentColor;
+    width: 1.0625rem;
+    height: ${GEL_SPACING_DBL};
+    margin: 0;
+  }
+`;
+
+const DurationWrapper = styled.span`
+  ${({ dir }) =>
+    dir === 'ltr'
+      ? `padding-left: ${GEL_SPACING};`
+      : `padding-right: ${GEL_SPACING};`}
+`;
+
+const programStateConfig = {
+  live: {
+    backgroundColor: C_POSTBOX,
+    headerLabelColor: C_POSTBOX,
+    headerTextColor: C_EBON,
+    titleColor: C_SHADOW,
+    durationColor: C_WHITE,
+  },
+  next: {
+    backgroundColor: C_WHITE,
+    headerLabelColor: C_KINGFISHER,
+    headerTextColor: C_METAL,
+    titleColor: C_METAL,
+    durationColor: C_KINGFISHER,
+  },
+  onDemand: {
+    backgroundColor: C_EBON,
+    headerTextColor: C_EBON,
+    titleColor: C_SHADOW,
+    durationColor: C_WHITE,
+  },
+};
+
+const renderHeaderContent = ({
+  type,
+  link,
+  translation,
+  brandTitle,
+  episodeTitle,
+  service,
+  script,
+  startTime,
+}) => {
+  const isOnDemand = type === 'onDemand';
+  const hiddenTextProps =
+    translation.toLowerCase() === 'live' ? { lang: 'en-GB' } : {};
+
+  const content = (
+    <>
+      {!isOnDemand && (
+        <VisuallyHiddenText {...hiddenTextProps}>
+          {`${translation}, `}
+        </VisuallyHiddenText>
+      )}
+      <VisuallyHiddenText>
+        {`${brandTitle}, ${startTime}, ${episodeTitle}`}
+      </VisuallyHiddenText>
+      {!isOnDemand && (
+        <LabelWrapper
+          service={service}
+          script={script}
+          {...programStateConfig[type]}
+        >
+          {`${translation.toUpperCase()} `}
+        </LabelWrapper>
+      )}
+      {brandTitle}
+      <TitleWrapper
+        service={service}
+        script={script}
+        {...programStateConfig[type]}
+      >
+        {episodeTitle}
+      </TitleWrapper>
+    </>
+  );
+
+  return type === 'next' ? content : <Link href={link}>{content}</Link>;
+};
+
+const ProgramCard = ({
+  dir,
+  service,
+  script,
+  brandTitle,
+  summary,
+  episodeTitle,
+  duration: { durationValue, durationText },
+  startTime,
+  state: { type, translation },
+  link,
+}) => (
+  <CardWrapper>
+    <TextWrapper>
+      <HeadingWrapper
+        service={service}
+        script={script}
+        {...programStateConfig[type]}
+      >
+        {renderHeaderContent({
+          type,
+          link,
+          translation,
+          brandTitle,
+          episodeTitle,
+          service,
+          script,
+          startTime,
+        })}
+      </HeadingWrapper>
+      <SummaryWrapper service={service} script={script}>
+        {summary}
+      </SummaryWrapper>
+    </TextWrapper>
+    <ButtonWrapper
+      service={service}
+      script={script}
+      {...programStateConfig[type]}
+    >
+      <IconWrapper {...programStateConfig[type]}>
+        {mediaIcons.audio}
+      </IconWrapper>
+      <DurationWrapper dir={dir}>
+        <VisuallyHiddenText>
+          {`${durationText} ${durationValue.replace(/:/g, ',')}`}
```

Comment:
Hmm... I'm wondering if we want to pass the duration in as a number and then format it in the component (so we could format in one way for presentation, and another for the screen reader text). Another thing I haven't considered before is do we format durations in the same way for all services? A question for Jon and the UX team maybe?